cmd/serf/command/agent/config.go

```diff
@@ -18,6 +19,7 @@ import (
 
 // This is the default port that we use for Serf communication
 const DefaultBindPort int = 7946
+const MaxNodeName int = 128
```

Comment:
This should probably reference the field in the serf package with the same name instead of defining it again here